index.rst

```diff
@@ -1,42 +1,621 @@
+
+.. math::
+  \boldsymbol{\Psi} = \boldsymbol{\phi}_\mathrm{dec}^T
+    {\bf C}_\mathrm{dec}^{-1} {\bf z}_\mathrm{dec}
+  :label: eq:decorrelated_coadd
+
+As the notation implies, this allows us to rewrite the joint log likelihood as
+
+.. math:: L = -\frac{1}{2}
+  \left[
+    {\bf z}_\mathrm{dec} - \boldsymbol{\phi}_\mathrm{dec} {\bf h}
+  \right]^T
+  {\bf C}_\mathrm{dec}^{-1}
+  \left[
+    {\bf z}_\mathrm{dec} - \boldsymbol{\phi}_\mathrm{dec} {\bf h}
+  \right]
+
+This identifies :math:`{\bf z}_\mathrm{dec}` as the decorrelated coadd image, :math:`{\bf C}_\mathrm{dec}` as its covariance matrix, and :math:`\boldsymbol{\phi}_\mathrm{dec}` as its PSF.  These can be used in exactly the same way as the corresponding single-exposure quantities.  As such, this is essentially the ideal coadd: it is formally optimal, can be used in the same way as any standard image, and makes no restrictive assumptions about the input images.
+
+The problem is of course the computational expense.  Despite the fact that we have derived the decorrelated coadd from the likelihood coadd, we do not necessarily need to produce a full likelihood coadd first; it may be possible to devise an algorithm that factors :math:`\boldsymbol{\Phi}` in small regions as it is constructed.  And the decorrelated coadd quantities :math:`{\bf C}_\mathrm{dec}` and :math:`\boldsymbol{\phi}_\mathrm{dec}` may be much more amenable to compression than :math:`\boldsymbol{\Phi}`.
+
+Because we have merely specified that :math:`{\bf C}_\mathrm{dec}` be "nearly" diagonal, this decomposition is not unique, and we have considerable flexibility to move power between :math:`\boldsymbol{\phi}_\mathrm{dec}` to make computation, storage, and use more efficient (without any change in the formal optimality).  Generally speaking, though, we want these quantities to mimic their standard image counterparts:
+
+- We want :math:`{\bf C}_\mathrm{dec}` to be close to diagonal, and to capture small-scale changes in the variance due to bright objects.
+- We want :math:`\boldsymbol{\phi}_\mathrm{dec}` to be compact and to vary smoothly (and slowly) over the image, to allow us to approximate the PSF as spatially constant over the scale of an object.
+
+These are generally competing goals, as can be seen from the limiting cases (which are not necessarily solutions, especially when :eq:`eq:decorrelated_coadd` is considered)
+
+.. math::
+  {\bf C}_\mathrm{dec} =& \, \boldsymbol{\Phi} \\
+  \boldsymbol{\phi}_\mathrm{dec} =& \, {\bf I}
+
+and
+
+.. math::
+  {\bf C}_\mathrm{dec} =& \, {\bf I} \\
+  \boldsymbol{\phi}_\mathrm{dec} =& \, \boldsymbol{\Phi}^{1/2}
```

Comment:
I think this one's still correct; what specifically do you think needs to be inverted?